### DISCUSSION TITLE
Should people pages contain links to personal pages?

### DISCUSSION THREAD

**Discussion by zouharvi** [machinetranslate/machinetranslate.org#247] [Q&A] [answered]
After writing a page for a researcher I was later asked to include a link to their personal website.

What are your thoughts on this? In my view:

Pros:
- It honors their wish
- Their personal website probably contains more information for those interested

Cons:
- Some researchers have multiple personal websites, some of which are maintained by the department, not updated often, etc
- The links also change often for some people when they change institutions

**Comment by cefoo** [comment]
To me, it's fine to include their personal websites - it would be somehow equivalent to including companies' official website, which we do (see [Companies](https://machinetranslate.org/companies)), but perhaps less evergreen.

I'm also thinking the following:

- They could link to us in their personal websites.
- They could update their Machine Translate article themselves by creating a PR, as companies do to be included in the Companies page.

Re the cons you listed, which I agree with 100%, maybe we can make a rule of including their personal website if they are really personal, and not a page within an institution. In that case, if the person changes institutions, the personal website would reflect their path.

**Reply by zouharvi** [reply]
Thanks for the reply.

> They could link to us in their personal websites.

If I was in their position, I wouldn't link to myself on something like a Wikipedia just because it would feel pretentious. :sweat_smile: But that's just my very personal feeling and I agree that the more people who link to Machinetranslate, the better.

I created a PR (#248) with a few links. When searching for the websites, it occured to me that even though many of them don't have any, some are *very* active on e.g. Twitter. I suppose that linking to their Twitter handle is even less robust than personal links?

**Comment by cefoo** [comment, marked as answer]
> it would feel pretentious

You are right! I didn't think of it that way. 

> I suppose that linking to their Twitter handle is even less robust than personal links?

Yeah, I wouldn't link to their Twitter accounts. I guess Twitter is mainly about spontaneity and what's happening at the moment of posting, (and liking cat memes, haha). It's also pretty easy to abandon an account.

Thank you for your PR! I'll review it later today!

**Comment by bittlingmayer** [comment]
I'd just add a bullet pointed list of links at the bottom of the page, after a horizontal rule.  We could almost make this part of the template, and then you just delete those that he or she doesn't have.

---

### See also
- [ personal website ]
- [ academic website ]
- [ GitHub ]
- [ something with citations like Google Scholar or Arxiv ]
- [ any news articles ] 
- [ Wikipedia article, if applicable ]
